pinot-broker/src/main/java/org/apache/pinot/broker/broker/helix/BaseBrokerStarter.java

```diff
@@ -237,25 +237,22 @@ public void start()
             Helix.DEPRECATED_ENABLE_CASE_INSENSITIVE_KEY, false);
     TableCache tableCache = new TableCache(_propertyStore, caseInsensitive);
     // Configure TLS for netty connection to server
-    TlsConfig tlsDefaults = TlsUtils.extractTlsConfig(_brokerConf, Broker.BROKER_TLS_PREFIX);
+    TlsConfig tlsDefaults = null;
+    if (_brokerConf.getProperty(Broker.BROKER_NETTYTLS_ENABLED, false)) {
```

Comment:
Right now Broker can have only one BrokerRequestHandler. We will use the same `Broker.BROKER_NETTYTLS_ENABLED` flag to enable gRPC server secure flag.